Makefile

```diff
@@ -46,6 +48,9 @@ yq:
 kustomize:
 	curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
 
+dashboard_linter:
+	go install github.com/grafana/dashboard-linter@latest
```

Comment:
It would be better no to use latest. We should use a static version.

Reply:
When I install the latest, it shows this prompt:
`go: downloading github.com/grafana/dashboard-linter v0.0.0-20230531105903-cd7fcaf3bec8`
We can use this version for our installation:
`go install github.com/grafana/dashboard-linter@v0.0.0-20230531105903-cd7fcaf3bec8`

Reply:
Thanks, @avi-biton! I changed it for your way of doing it now